addons/binding/org.openhab.binding.spotify/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,366 @@
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackHref" advanced="true">
+		<item-type>String</item-type>
+		<label>Track href</label>
+		<description>Channel reports track href currently played</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackUri" advanced="true">
+		<item-type>String</item-type>
+		<label>Track Uri</label>
+		<description>Channel reports track Uri currently played</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackName">
+		<item-type>String</item-type>
+		<label>Track Name</label>
+		<description>Channel reports currently played track name</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackType" advanced="true">
+		<item-type>String</item-type>
+		<label>Track Type</label>
+		<description>Channel reports currently played track type</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackDurationMs" advanced="true">
+		<item-type>Number</item-type>
+		<label>Track Duration ms</label>
+		<description>Channel reports currently played track duration (ms)</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackProgressMs" advanced="true">
+		<item-type>Number</item-type>
+		<label>Track Progress ms</label>
+		<description>Channel reports currently played track progress (ms)</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackDuration">
+		<item-type>String</item-type>
+		<label>Track Duration (m:ss)</label>
+		<description>Channel reports currently played track duration formatted (m:ss)</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackProgress">
+		<item-type>String</item-type>
+		<label>Track Progress (m:ss)</label>
+		<description>Channel reports currently played track progress formatted (m:ss)</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackNumber" advanced="true">
+		<item-type>Number</item-type>
+		<label>Track Number</label>
+		<description>Channel reports currently played track number</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackDiscNumber" advanced="true">
+		<item-type>Number</item-type>
+		<label>Track Disc Number</label>
+		<description>Channel reports currently played track disc number</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentlyPlayedTrackPopularity" advanced="true">
+		<item-type>Number</item-type>
+		<label>Track Popularity</label>
+		<description>Channel reports currently played track popularity</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="currentlyPlayedAlbumId" advanced="true">
+		<item-type>String</item-type>
+		<label>Album href</label>
```

Comment:
This label is a duplicate from the next ~~one~~ *two* and is not capitalised like the rest.